<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CropScoreboardWidget</class>
 <widget class="QWidget" name="CropScoreboardWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>897</width>
    <height>726</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="runButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>660</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
  </widget>
  <widget class="QLabel" name="uiTextLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>610</y>
     <width>871</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Press &quot;Run&quot; to start cropping the scoreboard</string>
   </property>
  </widget>
  <widget class="QLabel" name="videoLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>861</width>
     <height>561</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
